extensions/sql-migration/src/wizard/skuRecommendation/skuDataCollectionToolbar.ts

```diff
@@ -63,7 +84,47 @@ export class SkuDataCollectionToolbar implements vscode.Disposable {
 					...styles.TOOLBAR_CSS
 				}
 			}).component();
-		// TODO - implement onDidClick and add to disposables
+
+		this._disposables.push(startPerformanceCollectionButton.onDidClick(async () => {
+			const selectedOption = await vscode.window.showInformationMessage(
+				constants.AZURE_RECOMMENDATION_DATA_COLLECTION_POPUP_MESSAGE_LABEL,
+				constants.AZURE_RECOMMENDATION_DATA_COLLECTION_DEFAULT_PATH,
+				constants.AZURE_RECOMMENDATION_DATA_COLLECTION_CHOOSE_PATH
+			);
+
+			// Default path is selected or no option is selected.
+			if (!selectedOption || selectedOption === constants.AZURE_RECOMMENDATION_DATA_COLLECTION_DEFAULT_PATH) {
+				this._performanceDataSource = PerformanceDataSourceOptions.CollectData;
+				this.migrationStateModel._skuRecommendationPerformanceLocation = Default_PATH_FOR_START_DATA_COLLECTION;
+
+				// TODO - Start data collection at default path.
+			}
+			// 'Choose a path' option is selected.
+			else if (selectedOption === constants.AZURE_RECOMMENDATION_DATA_COLLECTION_CHOOSE_PATH) {
+				const options: vscode.OpenDialogOptions = {
+					openLabel: 'Select',
```

Comment:
Not sure if we need to localize this label 'Select', as this dialog is not part of ADS. But please check once from other references. I think it should be localized.

Reply:
I think we can keep it as localized. Also show something similar to be used in other extensions as well.